crates/wasm-encoder/src/reencode.rs

```diff
@@ -0,0 +1,1308 @@
+#![allow(missing_docs)] // FIXME
```

Comment:
Mind putting this on the `trait` and then it can be slowly filled in from there? Otherwise though I think it would be good to leave a high-level doc-comment on the module here indicating roughly what it's used for and perhaps a small example.

Reply:
Done